osgp/shared/osgp-ws-smartmetering/src/main/resources/schemas/sm-external-interface.xsd

```diff
@@ -1,10 +1,11 @@
-<!-- Copyright 2015 Smart Society Services B.V. Licensed under the Apache
+<!-- Copyright 2020 Smart Society Services B.V. Licensed under the Apache
   License, Version 2.0 (the "License"); you may not use this file except in
   compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0 -->
 <xsd:schema xmlns:xsd="http://www.w3.org/2001/XMLSchema"
             xmlns:bun="http://www.opensmartgridplatform.org/schemas/smartmetering/sm-bundle/2014/10"
             xmlns:con="http://www.opensmartgridplatform.org/schemas/smartmetering/sm-configuration/2014/10"
             xmlns:adm="http://www.opensmartgridplatform.org/schemas/smartmetering/sm-administration"
+            xmlns:tns="http://www.opensmartgridplatform.org/schemas/smartmetering/external-interface"
```

Comment:
This is not necessary. Must be kept in sync with targetNameSpace.

Reply:
> This is not necessary. Must be kept in sync with targetNameSpace.

I do need this, otherwise I cannot reference the DeviceType element.